branchpro/apps/dash_app.py

```diff
@@ -0,0 +1,48 @@
+#
+# Dash app
+#
+# This file is part of BRANCHPRO
+# (https://github.com/SABS-R3-Epidemiology/branchpro.git) which is released
+# under the BSD 3-clause license. See accompanying LICENSE.md for copyright
+# notice and full license details.
+#
+
```

Comment:
Somewhere at the top of this file, it would be useful to include some more details of what this app file actually does
- This is an app which shows forward simulation of the branching process model with fixed example data
- It can be run using $ python dash_app.py